ui/src/app/pages/access/modules-for-annual-renewal.tsx

```diff
@@ -25,13 +546,21 @@ export const ModulesForAnnualRenewal = (props: RenewalCardProps) => {
   };
   return (
     <FlexColumn style={styles.modulesContainer}>
-      {modules.map((moduleName) => {
+      {modules.map((moduleName, index) => {
         // TODO RW-7797.  Until then, hardcode
         const showModule = true;
         return (
           <FlexColumn
```

Comment:
existing problem, but when we have an array of components, we should supply a unique `key` prop to each to ensure that React renders them correctly.  Here I think it would be as simple as adding `key={index}` to this FlexColumn.  I expect `moduleName` would also work.

More info if you are not familiar: https://reactjs.org/docs/lists-and-keys.html